modules/installation-vsphere-machines.adoc

```diff
@@ -19,24 +19,49 @@ Before you install a cluster that contains user-provisioned infrastructure on VM
 
 .Procedure
 
-. Convert the control plane, compute, and bootstrap Ignition config files to Base64 encoding.
-
+. Locate the following Ignition config files that the installation program created, for example:
++
+* `<installation_directory>/bootstrap.ign`
+* `<installation_directory>/master.ign`
+* `<installation_directory>/worker.ign`
++
+. Convert the bootstrap Ignition config file to Base64 encoding and store it in the extra configuration properties of the virtual machine (VM) before booting.
 +
-For example, if you use a Linux operating system, you can use the `base64` command to encode the files.
+* To convert to Base64 and change the bootstrap Ignition config file, enter the following commands:
++
+[source,terminal]
+----
+$ govc vm.change -vm "<vm_name>" -e "guestinfo.ignition.config.data.encoding=base64"
+----
++
+[source,terminal]
+----
+$ govc vm.change -vm "<vm_name>" -e "guestinfo.ignition.config.data=$(base64 -w0 bootstrap.ign)"
```

Comment:
This doesn't work because of bash length limitations.
```
[jcallen@fedora vsphere-upi]$  govc vm.change -vm "jcallen-registry" -e "guestinfo.ignition.config.data=$(base64 -w0 bootstrap.ign)"
bash: /home/jcallen/bin/govc: Argument list too long
[jcallen@fedora vsphere-upi]$
```
PowerShell (w/PowerCLI) does though
```
$vm | New-AdvancedSetting -name "guestinfo.ignition.config.data" -value $Ignition -confirm:$false -Force > $null
```
PowerCLI is probably more used by VMware administrators than govc. (I have no data to back this up, just that PowerCLI has been around for significantly longer period of time)

Reply:
Thanks, @jcpowermac. Is the PowerCLI command necessary to run separately for bootstrap, master, and worker nodes? (If so, how does it distinguish each?)

Or do they _not_ need step 1 and for step 2, do they only need to run the command once like below?

> ~2~1. Store the Ignition config file in the extra configuration properties of the virtual machine (VM) before booting by running the PowerCLI (`vm`) command:
> 
> `$ vm | New-AdvancedSetting -name "guestinfo.ignition.config.data" -value $Ignition -confirm:$false -Force > $null`